docs/scenarios/demo.md

```diff
@@ -192,29 +192,31 @@ Due to the absence of support for Remote EJBs in Quarkus, you will notice that t
 
 ![MDB - Manual updates](mdbchanges.png)
 
-### 2.3 Deploy app to OpenShift
+### 2.3 Deploy app to Kubernetes
 
-Now, its time to deploy the coolstore quarkus app on OpenShift cluster.
+Although the app is deployable to any [Kubernetes](https://kubernetes.io/) distribution. 
+For the sake of simplicity we choose [minikube](https://minikube.sigs.k8s.io/docs/). 
 
-- Make sure you can access the cluster from your terminal
-- Navigate to the coolstore project folder
-- Deploy a postgres container
+> Prerequisite: It is assumed that minikube is installed. If not you can follow the instructions [here](https://minikube.sigs.k8s.io/docs/start/?arch=%2Fmacos%2Fx86-64%2Fstable%2Fbinary+download).
```

Comment:
should we just point to the install docs - https://minikube.sigs.k8s.io/docs/start/ and let it auto select the arch?

Reply:
ACK. now changed